x/consensus_param/migrations/v2/store.go

```diff
@@ -0,0 +1,45 @@
+package v2
+
+import (
+	"github.com/cosmos/cosmos-sdk/codec"
+	storetypes "github.com/cosmos/cosmos-sdk/store/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/consensus_param/exported"
+	v1 "github.com/cosmos/cosmos-sdk/x/consensus_param/migrations/v1"
+	"github.com/cosmos/cosmos-sdk/x/consensus_param/types"
+	tmproto "github.com/tendermint/tendermint/proto/tendermint/types"
+
+	tmtypes "github.com/tendermint/tendermint/types"
+)
+
+func MigrateStore(ctx sdk.Context, storeKey storetypes.StoreKey, cdc codec.BinaryCodec, legacySubspace exported.ParamStore) error {
+	store := ctx.KVStore(storeKey)
+
+	consensusParams := new(tmproto.ConsensusParams)
+
+	consensusParams.Version = &tmproto.VersionParams{AppVersion: 0}
```

Comment:
@marbar3778 I'm not sure how to handle the version for now I left the same value that is in default params, but maybe when you see this we can discuss it

Reply:
this would be the app version set by the upgrade module I believe. We may need to get it from there or leave it blank for now

Reply:
I left an empty struct. However, I'm reading through x/upgrade to check if I should bring it from there

Reply:
Should I just pass the upgrade keeper as a dependency?